include/aws/http/private/websocket_decoder.h

```diff
@@ -42,7 +42,7 @@ struct aws_websocket_decoder {
     /* True while processing a TEXT "message" (from the start of a TEXT frame,
      * until the end of the TEXT or CONTINUATION frame with the FIN bit set). */
     bool processing_text_message;
-    struct aws_utf8_validator *text_message_validator;
```

Comment:
trivial: We should rename the variable as well.

Reply:
keep it as is.
I realized they're different, but I'm still only doing validation here, so I think this name is better